### PR TITLE
Coopr 589:Document SSL/TLS usage for coopr-standalone

### PR DESCRIPTION
**Pull request by artemmakogon**
@jwang47, @wolf31o2  please review

### CODE REVIEW

File: coopr-docs/docs/source/installation/provisioner-config.rst [hunk 1]
```diff
@@ -58,6 +58,27 @@ Shown below is an example configuration with settings that you are most likely t
...
+      <description>URI of server to connect to</description>
+    </property>
+
+To configure the provisioner to support server that uses mutual authentication with SSL,
+setup these environment variables:
+
+====================================     ==========================    =======================================
+   Environment variable                     Default Value                     Description
+====================================     ==========================    =======================================
+TRUST_CERT_PATH                             None                        Trusted certificate file location.
```
Comment (jwang47) [inline coopr-docs/docs/source/installation/provisioner-config.rst:78]: What's the difference between trusted and not trusted?

Comment (wolf31o2) [inline coopr-docs/docs/source/installation/provisioner-config.rst:78]: Here's the problem. The provisioner, itself, should **only** ever read from its own configuration file. There should be configuration options in the `provisioner-site.xml` for this. For standalone, we should modify the standalone-specific `provisioner-site.xml` located at `coopr-standalone/provisioner/master/conf/provisioner-site.xml` with some placeholder variables and sed them, like we do for the `provisioner.data.dir` and `provisioner.work.dir` properties.

File: coopr-standalone/bin/coopr.sh [hunk 2]
```diff
@@ -164,8 +164,6 @@ fi
 
 export SECURITY_ENABLED=`read_property security.enabled ${COOPR_SERVER_CONF}/coopr-site.xml`
 export COOPR_SERVER_URI=${COOPR_SERVER_URI:-${COOPR_PROTOCOL}://127.0.0.1:55054}
-export TRUST_CERT_PATH=`read_property server.ssl.trust.cert.path ${COOPR_SERVER_CONF}/coopr-security.xml`
-export TRUST_CERT_PASSWORD=`read_property server.ssl.trust.cert.password ${COOPR_SERVER_CONF}/coopr-security.xml`
```
Comment (wolf31o2) [inline coopr-standalone/bin/coopr.sh:168]: Why is this in a documentation PR?

File: coopr-docs/docs/source/installation/provisioner-config.rst [hunk 3]
```diff
@@ -58,6 +58,27 @@ Shown below is an example configuration with settings that you are most likely t
...
+TRUST_CERT_PASSWORD                         None                        Trusted certificate password.
```
Comment (wolf31o2) [inline coopr-docs/docs/source/installation/provisioner-config.rst:79]: This belongs in `provisioner-security.xml` which means support needs to be added for this new file.

File: coopr-docs/docs/source/installation/ui-config.rst [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+Running UI with SSL
+^^^^^^^^^^^^^^^^^^^^^^^
+
+To enable running UI with SSL, setup these environment variables:
+
+====================================     ==========================    =======================================
+   Environment variable                     Default Value                     Description
+====================================     ==========================    =======================================
+COOPR_UI_KEY_FILE                        COOPR_HOME/cert/server.key     Key file location.
+COOPR_UI_CERT_FILE                       COOPR_HOME/cert/server.crt     Certificate password.
```
Comment (wolf31o2) [inline coopr-docs/docs/source/installation/ui-config.rst:34]: Why do these defaults exist if they're invalid? They point to nonexistent locations, and there's no way to properly enable/disable SSL support using this.

**Comment by wolf31o2** [comment]
We will have to do a new request for this, since it doesn't apply.

**Pull request closed without merging**